backend/src/test/java/com/allog/dallog/common/fixtures/ScheduleFixtures.java

```diff
@@ -36,7 +36,7 @@ public class ScheduleFixtures {
     public static final ScheduleCreateRequest 알록달록_회의_생성_요청 = new ScheduleCreateRequest(알록달록_회의_제목, 알록달록_회의_시작일시,
             알록달록_회의_종료일시, 알록달록_회의_메모);
     public static final ScheduleResponse 알록달록_회의_응답 = new ScheduleResponse(1L, 1L, 알록달록_회의_제목, 알록달록_회의_시작일시,
-            알록달록_회의_종료일시, 알록달록_회의_메모);
+            알록달록_회의_종료일시, 알록달록_회의_메모, "NORMAL");
```

Comment:
```suggestion
            알록달록_회의_종료일시, 알록달록_회의_메모, CategoryType.NORMAL.name());
```

CategoryType Enum에서 문자열 값을 가져오는 건 어떨까요?